include/ipc/ipc_service.h

```diff
@@ -66,22 +87,29 @@ extern "C" {
 struct ipc_service_cb {
 	/** @brief Bind was successful.
 	 *
-	 *  @param priv Private user data.
+	 *  This callback is called when the endpoint binding is successful.
+	 *
+	 *  @param[in] priv Private user data.
 	 */
 	void (*bound)(void *priv);
 
 	/** @brief New packet arrived.
 	 *
-	 *  @param data Pointer to data buffer.
-	 *  @param len Length of @a data.
-	 *  @param priv Private user data.
+	 *  This callback is called when new data is received.
+	 *
+	 *  @note The data buffer is to be considered released and available
+	 *        again only when this callback returs.
```

Comment:
s/returs/returns/

Reply:
yeah, that was fixed by a following commit in the same PR.